wpimath/src/main/java/edu/wpi/first/math/ComputerVisionUtil.java

```diff
@@ -16,7 +16,7 @@ private ComputerVisionUtil() {
   }
 
   /**
-   * Algorithm from https://docs.limelightvision.io/en/latest/cs_estimating_distance.html Estimates
+   * Algorithm from <a href="https://docs.limelightvision.io/en/latest/cs_estimating_distance.html">Limelight</a> Estimates
```

Comment:
This should be done in ComputerVisionUtil.h as well. "Algorithm from Limelight Estimates" doesn't make much sense. "Algorithm from the Limelight docs that estimates" would be better.